### PR TITLE
Moving content from composer to use-cases

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Description
Moved the examples from the Composer page to the Use-Cases section and arranged the remaining content on the Composer page to make it logically ordered

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/use-cases/auth-token.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+As part of good testing strategy is create end-to-end tests that resemble common user flows. Let's consider the scenario where a company has an authentication server. This server, when given the proper user credentials, returns an authentication token. This token is required for all other calls throughout the platform's API environment. Without this first API call, none of the other API calls can work.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+## Token-based Authentication API
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:17]:
```suggestion
```

File: docs/api-testing/use-cases/auth-token.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+Now, we'll show how to make the authentication call, save the returned token in a variable and use it to make subsequent calls.
+
+### Step 1: Make the authorization call
+
+As first step we need to make the call to retrive the token.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:23]:
```suggestion
First, make the call to retrieve the token.
```

File: docs/api-testing/use-cases/auth-token.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+<img src={useBaseUrl('/img/api-testing/int-token.png')} alt="The user token" width="400"/>
+
+:::tip
+You can double-check that the response contains the correct data by adding an assertion verifying that the `user_id` provided in the Request Body is the same as in the Response Payload.
+:::
+
+### Step 2: Save the token for further calls
+
+Next, we need to assign the token to a variable. Variables are used to store data temporarily for a test, but you can use the Sauce Labs API Testing Vault for permanent variables. For more information, see [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault)).
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:55]:
```suggestion
Next, assign the token to a variable. Variables are used to store data temporarily for a test, but you can use the Sauce Labs API Testing Vault for permanent variables. For more information, see [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault)).
```

File: docs/api-testing/use-cases/auth-token.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+This step takes the `access_token` value in the `authPayload` response, and sets it as `access_token`. The response body from the orginal POST call was saved to a variable called `authPayload` and the key for the token is `access_token`, which can be found by calling `authPayload.access_token`. We do this so we don't have to manually invoke a variable every time it is needed.
+
+:::note
+The dollar sign and brackets are necessary when referencing variables so that Sauce Labs API Testing knows to interpret what’s between the brackets instead of using it literally.
+:::
+
+### Step 3: Make a follow-up call
+
+Next, as third step we are going to call the cart function that requires the token granted by the authentication server in order to add items to a cart.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:73]:
```suggestion
Now you can call the cart function that requires the token granted by the authentication server to add items to a cart.
```

File: docs/api-testing/use-cases/auth-token.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+    "9": 3
+}
+```
+
+Now, you can add assertions to verify the payload response returned by the call.
+
+### Step 4: Reuse the token in one more call
+
+You can reuse the token as many times as you want inside the same test.
+As last step, we are going to call the cart to view items currently into it, after the update.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:104]:
```suggestion
Now that you have updated the cart, you can add a `GET` request to return the items that are currently in it. 
```

File: docs/api-testing/use-cases/auth-token.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+## Token-based Authentication API
+
+Now, we'll show how to make the authentication call, save the returned token in a variable and use it to make subsequent calls.
+
+### Step 1: Make the authorization call
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:19]:
```suggestion
### Step 1: Make the Authorization Call
```

File: docs/api-testing/use-cases/auth-token.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+
+<img src={useBaseUrl('/img/api-testing/int-assign-token.png')} alt="Setting the variable"/>
+
+This step takes the `access_token` value in the `authPayload` response, and sets it as `access_token`. The response body from the orginal POST call was saved to a variable called `authPayload` and the key for the token is `access_token`, which can be found by calling `authPayload.access_token`. We do this so we don't have to manually invoke a variable every time it is needed.
+
+:::note
+The dollar sign and brackets are necessary when referencing variables so that Sauce Labs API Testing knows to interpret what’s between the brackets instead of using it literally.
+:::
+
+### Step 3: Make a follow-up call
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:69]:
```suggestion
### Step 3: Make a Follow-up Call
```

File: docs/api-testing/use-cases/auth-token.md [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+
+Given proper credentials, the authentication server will return a payload response with `user_id` and `access_token`
+
+<img src={useBaseUrl('/img/api-testing/int-token.png')} alt="The user token" width="400"/>
+
+:::tip
+You can double-check that the response contains the correct data by adding an assertion verifying that the `user_id` provided in the Request Body is the same as in the Response Payload.
+:::
+
+### Step 2: Save the token for further calls
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/auth-token.md:51]:
```suggestion
### Step 2: Save the Token for Further Calls
```

File: docs/api-testing/use-cases/integration-test.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+One of the core features of the platform is the ability to create proper Integration Tests. An Integration Test is a test in which you examine a complete user flow, simulating what an API consumer would experience. Integration testing is critical for creating a strong API testing strategy. While only testing individual endpoints is a good start, this method will miss a large number of problems that occur when all services need to work together.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+## Test Interactions between Endpoints
+
+For the purposes of this guide, will keep our example simple and work only with two API endpoints but your integration test can be longer.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/integration-test.md:19]:
```suggestion
For the purposes of this guide, the example is simple and works with two API endpoints, but your integration test can be longer.
```

File: docs/api-testing/use-cases/integration-test.md [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+In the following example, we'll use an API endpoint that produces an array of all the available products (http://demoapi.apifortress.com/api/retail/product) and another endpoint that shows the details of a specific product based on its ID (http://demoapi.apifortress.com/api/retail/product/${id}).
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/integration-test.md:20]:
```suggestion
In the following example, uses an API endpoint that produces an array of all the available products (http://demoapi.apifortress.com/api/retail/product) and another endpoint that shows the details of a specific product based on its ID (http://demoapi.apifortress.com/api/retail/product/${id}).
```

File: docs/api-testing/use-cases/integration-test.md [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+4. Test the response payload for the endpoint by adding assertions.
+
+5. Add a new `Set(variable)` assertion to set the `id` variable as every single product that is returned:
+   - Variable (the variable name) - for example `id`
+   - Mode (the variable type) - for example `String`
+   - Value (the value the variable will contain) - for example `${_1.id}`
+
+The system uses `_1` automatically when recognizing a subroutine, which makes it easier when there are multiple sub-levels.
+
+Below how your test might look like till the above step:
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/integration-test.md:52]: Not sure what you're trying to say here. I tried fix it. 
```suggestion
This is how your test might look after you set the `id` variable:
```

**Comment by LoryApiFortress** [comment]
@aliciakrebs I made all the changes you suggested but I cannot merge, can you please approve?

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**